NativeImport/ExampleNativeImportFeed.xml

```diff
@@ -79,6 +63,143 @@
                 <Featured>false</Featured>
                 <DisplayLocale>en_US</DisplayLocale>
             </Review>
+
+            <!-- sample review featuring attributes -->
+
+            <Review id="11" removed="false">
+                <ModerationStatus>APPROVED</ModerationStatus>
+                <UserProfileReference id="miketester112">
+                    <ExternalId>miketester112</ExternalId>
+                    <DisplayName>supermike</DisplayName>
+                    <Anonymous>false</Anonymous>
+                    <HyperlinkingEnabled>false</HyperlinkingEnabled>
+                </UserProfileReference>
+                <Title>Best product ever!</Title>
+                <ReviewText>This product truly changed my life; I don't know what I'd do without it.</ReviewText>
+                <Rating>5</Rating>
+                <IpAddress>64.94.228.137</IpAddress>
+                <Recommended>true</Recommended>
+                <UserEmailAddress>supermike@example.com</UserEmailAddress>
+                <ReviewerLocation>Austin, TX</ReviewerLocation>
+                <SubmissionTime>2012-01-23T12:59:25.000-06:00</SubmissionTime>
+                <Featured>false</Featured>
+                <DisplayLocale>en_US</DisplayLocale>
+
+                <!-- review feedbacks: -->
+
+                <NumFeedbacks>3</NumFeedbacks> <!-- Number of any feedbacks on a given review (positive + negative) -->
+                <NumPositiveFeedbacks>2</NumPositiveFeedbacks> <!-- Number of positive feedbacks on a given review -->
+                <NumNegativeFeedbacks>1</NumNegativeFeedbacks>  <!-- Number of negative feedbacks on a given review -->
+                <NumInappropriateFeedbacks>1</NumInappropriateFeedbacks> <!-- Number of times customers reported that this review has inappropriate or irrelevant content -->
+
+                <!-- add a custom question; create submission form question in configuration hub prior to import -->
+
+                <AdditionalFields>
+                      <AdditionalField id="CustomQuestion">
+                        <DisplayLabel>Custom Question:</DisplayLabel>
```

Comment:
Actually, fix indentation everywhere. There are a few different places where indentation is not correct. Please use four spaces for each level of indentation.